pkg/storage/etcd3/store.go

```diff
@@ -58,19 +61,25 @@ type objState struct {
 }
 
 // New returns an etcd3 implementation of storage.Interface.
-func New(c *clientv3.Client, codec runtime.Codec, prefix string) storage.Interface {
-	return newStore(c, codec, prefix)
+func New(c *clientv3.Client, quorumRead bool, codec runtime.Codec, prefix string) storage.Interface {
```

Comment:
This flag is mainly used for testing (or special environment when user knows they are running 1 member etcd cluster), and it requires a recompile. I do not really like to use bool as arg and want to keep New as simple as possible. So can we not modify the New func to keep it clean, and add another NewWithNoQuorumRead func?